Classes/EventListener/Mail/AttachmentFromOrderItem.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Extcode\Cart\EventListener\Mail;
+
+/*
+ * This file is part of the package extcode/cart.
+ *
+ * For the full copyright and license information, please read the
+ * LICENSE file that was distributed with this source code.
+ */
+
+use Extcode\Cart\Event\Mail\AttachementEvent;
+use TYPO3\CMS\Core\Utility\GeneralUtility;
+use TYPO3\CMS\Extbase\Configuration\ConfigurationManager;
+use TYPO3\CMS\Extbase\Persistence\ObjectStorage;
+
+class AttachmentFromOrderItem
+{
+    private array $pluginSettings;
+
+    public function __construct(
+        private ConfigurationManager $configurationManager
+    ) {
+        $this->pluginSettings = $this->configurationManager->getConfiguration(
```

Comment:
For me `pluginSettings` is somehow irritating. My first thought was that it's the settings from the BE plugin which is set via flexform. In my opinion only `settings` would be clearer. Or did I miss something?